boards/arm/nrf5340_audio_dk_nrf5340/Kconfig

```diff
@@ -0,0 +1,116 @@
+# nRF5340 Audio DK board configuration
+
+# Copyright (c) 2019 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config IPM_NRFX
+	default IPM
+
+config MBOX_NRFX_IPC
+	default MBOX
+
+config RPMSG_SERVICE_DUAL_IPM_SUPPORT
+	default RPMSG_SERVICE
+
+if RPMSG_SERVICE_DUAL_IPM_SUPPORT
+
+config IPM_MSG_CH_0_ENABLE
+	default y
+
+config IPM_MSG_CH_1_ENABLE
+	default y
+
+config RPMSG_SERVICE_IPM_TX_NAME
+	default "IPM_0" if RPMSG_SERVICE_MODE_MASTER
+	default "IPM_1" if RPMSG_SERVICE_MODE_REMOTE
+
+config RPMSG_SERVICE_IPM_RX_NAME
+	default "IPM_1" if RPMSG_SERVICE_MODE_MASTER
+	default "IPM_0" if RPMSG_SERVICE_MODE_REMOTE
+
+config IPM_MSG_CH_0_TX
+	default RPMSG_SERVICE_MODE_MASTER
+
+config IPM_MSG_CH_0_RX
+	default RPMSG_SERVICE_MODE_REMOTE
+
+config IPM_MSG_CH_1_TX
+	default RPMSG_SERVICE_MODE_REMOTE
+
+config IPM_MSG_CH_1_RX
+	default RPMSG_SERVICE_MODE_MASTER
+
+endif # RPMSG_SERVICE_DUAL_IPM_SUPPORT
+
+if BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP || BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP_NS
+
+config BOARD_ENABLE_DCDC_APP
+	bool "Enable Application MCU DCDC converter"
+	select SOC_DCDC_NRF53X_APP
+	default y
+
+config BOARD_ENABLE_DCDC_NET
+	bool "Enable Network MCU DCDC converter"
+	select SOC_DCDC_NRF53X_NET
+	default y
+
+config BOARD_ENABLE_DCDC_HV
+	bool "Enable High Voltage DCDC converter"
+	select SOC_DCDC_NRF53X_HV
+	default y
+
+choice BT_HCI_BUS_TYPE
+	default BT_RPMSG if BT
+endchoice
+
+config HEAP_MEM_POOL_SIZE
+	default 4096 if BT_RPMSG
```

Comment:
many of these entries are changing defaults, so please use `Kconfig.defconfig` file for that (see the nrf5340 board upstream, it was recently fixed)

Reply:
As far as I can see, those changes haven't been upmerged to NCS yet, so these settings are still in Kconfig for nrf5340 board

Reply:
right, but this is a new out-of-tree board, so it should be done well here.